docs/b2b-edition/about/index.mdx

```diff
@@ -48,4 +48,4 @@ For example, passing the following query param string returns the 11th through 2
 
 BundleB2B was a purpose-built application that powered BigCommerce's original B2B Edition offering.
 
-In 2021, we acquired BundleB2B. Since that time, we've been integrating the BundleB2B API and features into our core product offerings. We continue to use the BundleB2B domain to prevent breaking changes for existing integrations. Our roadmap includes plans to use a new core BigCommerce domain to host our B2B Edition APIs.
+In 2021, BigCommerce acquired BundleB2B. Since that time, we've been integrating the BundleB2B API and features into our core product offerings. We continue to use the BundleB2B domain to prevent breaking changes for existing integrations. Our roadmap includes plans to use a new core BigCommerce domain to host our B2B Edition APIs. `{*Does this need updating?*}`
```

Comment:
@bigcommerce/team-b2b should we update this after we complete our migration validation?